lib/search-field/style.scss

```diff
@@ -22,6 +22,10 @@
     flex: 0 0 auto;
   }
 
+  .icon-search-small {
+    max-width: 16px;
+  }
```

Comment:
I think what we actually want here is a `flex: 0 0 auto` which will tell it not to resize itself, but if you tried that and it didn't work, ignore me.

Reply:
I tried this and a few variations but still couldn't get the sizing right. It did lead me to look at what I think is a better option for setting the size instead of having to do it in multiple places. I've updated the approach now.